packages/test-base/src/jvmTest/kotlin/io/realm/kotlin/test/compiler/PersistedNameTests.kt

```diff
@@ -67,8 +67,6 @@ class PersistedNameTests {
         )
 
         assertEquals(KotlinCompilation.ExitCode.OK, result.exitCode)
-        val compilerLog = result.messages
```

Comment:
I guess the whole test should be removed. The intention is now caught by https://github.com/realm/realm-kotlin/pull/1246/files#diff-15e1aebddb1a90c5b093c9072231a81488ac5bcb71f580a96fb476951aed6d55R391 and this test name is completely off compared to what it actually does.